### ISSUE TITLE
Build CLI for migrations

### ISSUE THREAD

**Issue by lollar** [superposition-dev/lol-rm-migrations#3]
We need to write either several small CLI's for migrations for example...
```
create_table --name=blah # etc.
create_database --env=local
alter_table --name=blah # etc.
``` 
Or if we have one class to rule them all... like:
```
migrations --create-table # add in other variables
```
Need to grok usability for the CLI

**Comment by reecebisel** [comment]
Rather than passing in `--` arguments I think I'd like to see something more along the lines of:
```
lol-migration alter_table add_blah_to_bleep_bloop
lol-migration create_database
lol-migration create_table opinions
```
what are your thoughts on this?

**Comment by lollar** [comment]
Yeah, I like that more.  The one thing I want to avoid is trying to get implied meaning from what a user types in, I'd rather force them to be explicit. So alter_table would set up the structure for them but wouldn't try to derive column or table names from the add_blah_to_bleep_bloop... unless you can figure out a way to do it cleanly.

**Comment by reecebisel** [comment]
the `add_blah_to_bleep_bloop` part is the file name in my mind. Because I timestamp is great and all, but i need something to let me know what the hell that thing is going to do.

**Comment by lollar** [comment]
One thing I didn't consider with this is how something like `lol-migration run migrations` would keep track of which migrations need to be run.

**Comment by reecebisel** [comment]
Basically there would have to be some sort of check to see if the database had which tables with the appropriate columns. No clue how to do that off the top of my head.
